spring-kafka/src/main/java/com/baeldung/spring/kafka/streams/CustomValue.java

```diff
@@ -0,0 +1,18 @@
+package com.baeldung.spring.kafka.streams;
+
+public class CustomValue {
+
+    private String name;
+
+    public CustomValue(final String name) {
```

Comment:
Let's avoid this unnecessary `final`. We should only use `final` in our example code when it's essential, or making a point. In this instance, there's no benefit of `final`. We don't make the `String` any more or less immutable by doing it, and the constructor wouldn't be expected to have a risk of modifying its input parameters.

Reply:
Done